cclib/parser_properties/base_parser.py

```diff
@@ -7,3 +7,14 @@ class base_parser(ABC):
     @abc.abstractmethod
     def parse(file_handler, program, ccdata):
         return
+
+    @staticmethod
+    def check_dependencies(dependency_list, ccdata, current_property):
+        if len(dependency_list) > 0:
```

Comment:
You can get rid of this clause, line 14 is enough:
```
In [1]: mylist = []

In [2]: for i in mylist:
   ...:     print(f"hello, {i}!")
   ...:

In [3]:
```

Reply:
Addressed this, thanks!